plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/itemlist/ObjectListControl.java

```diff
@@ -1479,6 +1479,29 @@ private String copyGridToText() {
         @Override
         public void fillConfigMenu(IContributionManager menuManager) {
             super.fillConfigMenu(menuManager);
+            // Traverse all columns, if there is a column with name "#", add a
+            // menu option "Revert sorting"
+            // which is to sort in the ascending order of the "#" column
+            int columnsCount = this.getColumnsCount();
+            for (int i = 0; i < columnsCount; i++) {
+                if (this.getColumnName(i).equals("#")) {
+                    final int numberColumnInd = i;
```

Comment:
Why do you need this variable? I suppose you can just pass i as an argument.

Reply:
It will raise the error "Local variable i defined in an enclosing scope must be final or effectively final".